articles/is-innersource-right.md

```diff
@@ -0,0 +1,33 @@
+# Is InnerSource Right for My Project?
+
+InnerSource is the application of open source principles to company-internal software development. Done right, it unblocks progress and eases adoption of shared services and modules. This article contains guidance and questions to ask yourself when considering adoption of an InnerSource approach to running your project.
+
+## Will Contributions Come?
+
+An InnerSource approach only makes sense if contributions are expected from the project's users. You can expect contributions to come if you see or anticipate noticeable amounts of energy directed at your project space by its users. Some examples:
+
+-   High amounts of project usage and adoption.
+-   More feature requests than your team has time to fill.
+-   Users doing workarounds to compensate for lack of features in your project.
+-   Feature requests that take nearly as much time to explain as they would just to implement.
+-   Multiple roadmap dependencies on your project.
+
+## Will I Support Contributions?
+
+Even with willing contributors, the code doesn't just flow in. You will need to encourage and support contributions via activities such as:
+
+-   Understanding users' scenarios and suggesting what contributions in your project could help them to meet those scenarios.
+-   Inviting users to make the contributions that they need and following up with them to ensure that they do them.
+-   Maintaining an CONTRIBUTING.md document that contains everything an engineer needs to know to contribute to the project.
```

Comment:
The is an opportunity link to even more patterns here, if that is desirable.
e.g. https://patterns.innersourcecommons.org/p/base-documentation#contributing.md